migrations/remove-empty-parents.js

```diff
@@ -0,0 +1,55 @@
+var async = require('async'),
+    db = require('../db');
+
+function hasProperties(obj) {
+  var k;
+
+  if (!obj) {
+    return false;
+  }
+
+  for(k in obj) {
+    if(obj.hasOwnProperty(k)) {
+      return true;
+    }
+  }
+
+  return false;
+}
+
+function removeDeadParents(row, callback) {
+  console.log('Should process parent: ', row);
+  if(hasProperties(row.doc.parent)) {
+    return callback();
+  }
+  db.medic.get(row.id, function(err, doc) {
+    if(err) {
+      return callback(err);
+    }
+    delete doc.parent;
+    db.medic.insert(doc, function(err) {
```

Comment:
It looks like you can just pass the `callback` instead of a new anonymous function.